main.py

```diff
@@ -97,18 +172,12 @@ def draw(drawing: Drawing, single_shape: bool = False) -> Shape:
 def draw_third_stage(drawing: Drawing) -> Shape:
     """Draw the next round screen and return the chosen shape"""
     drawing.clear_screen()
-    path = './IAPS/*.jpg'
-
-    IAPS = []
-    for filename in glob.glob(path):
-        IAPS.append(filename)
-
     random.shuffle(IAPS)
     figure = IAPS.pop()
-
+    plus = get_iap(figure)
```

Comment:
I don't really know what plus means, I would consider naming it something more descriptive.